src/main/java/com/google/javascript/gents/TypeConversionPass.java

```diff
@@ -75,21 +82,46 @@ public void visit(NodeTraversal t, Node n, Node parent) {
         case CONST:
           // Converts a typedef into an interface, which then later has its members converted in
           // TypeAnnotationPass.
+          boolean needToExport = false;
           JSDocInfo jsdoc = NodeUtil.getBestJSDocInfo(n);
+
           if (jsdoc != null && jsdoc.hasTypedefType()) {
             String name;
             if (n.getToken() == Token.NAME) {
               name = n.getString();
             } else if (n.getToken() == Token.GETPROP) {
+              // Inner typedef.
+              // TypeScript doesn't have nested type aliases inside a class. Therefore this is moved to the top level.
+              needToExport = true;
               name = n.getSecondChild().getString();
             } else {
               name = n.getFirstChild().getString();
             }
             Node typeDef = Node.newString(Token.TYPE_ALIAS, name);
             types.put(name, typeDef);
             typeDef.setJSDocInfo(jsdoc);
+            if (needToExport) {
+              typesToRename.put(
+                  n.getFirstChild().getString() + "." + n.getSecondChild().getString(), name);
```

Comment:
No they are the same.
I was overwhelmed by the number of available APIs and didn't find this one.

Fixed.